画像読み込み処理でエラー

```
Traceback (most recent call last):
  File "/layers/google.python.pip/pip/bottle.py", line 868, in _handle
    return route.call(**args)
  File "/layers/google.python.pip/pip/bottle.py", line 1748, in wrapper
    rv = callback(*a, **ka)
  File "/srv/main.py", line 55, in upload_post
    im1 = cv2.imdecode(get_np_array(file1.file), 1)
cv2.error: OpenCV(4.2.0) /io/opencv/modules/imgcodecs/src/loadsave.cpp:730: error: (-215:Assertion failed) !buf.empty() in function 'imdecode_'
```

--- a/main.py
+++ b/main.py
@@ -1,3 +1,4 @@
+import io
 import base64
 import logging
 from pathlib import Path
@@ -36,6 +37,18 @@ def makeup(result_dict):
     return '-'.join(['{}{}'.format(k, v) for k, v in result_dict.items()])
 
 
+def is_valid_file(f):
+    if f is None:
+        logger.warning('file is not specified')
+        return False
+    f.seek(0, io.SEEK_END)
+    if f.tell() == 0:
+        logger.warning('blank file')
+        return False
+    f.seek(0)
+    return True
+
+
 @app.get('/upload')
 def upload_get():
     redirect('/')
@@ -46,7 +59,12 @@ def upload_post():
     file1 = request.files.get('file1')
     file2 = request.files.get('file2')
 
-    if file1 is None or file2 is None:
+    logger.info('test file1')
+    if not is_valid_file(file1.file):
+        redirect('/')
+
+    logger.info('test file2')
+    if not is_valid_file(file2.file):
         redirect('/')
 
     dropitems = img2str.DropItems(storage=storage)

--- a/static/index.html
+++ b/static/index.html
@@ -30,11 +30,11 @@
             <form method="post" action="/upload" enctype="multipart/form-data">
                 <div class="row responsive-label">
                     <div class="col-sm-12 col-md-2 label"><label for="before">周回前</label></div>
-                    <div class="col-sm-12 col-md"><input type="file" id="before" name="file1"></div>
+                    <div class="col-sm-12 col-md"><input type="file" id="before" name="file1" accept=".png,.jpg,.jpeg"></div>
                 </div>
                 <div class="row responsive-label">
                     <div class="col-sm-12 col-md-2 label"><label for="after">周回後</label></div>
-                    <div class="col-sm-12 col-md"><input type="file" id="after" name="file2"></div>
+                    <div class="col-sm-12 col-md"><input type="file" id="after" name="file2" accept=".png,.jpg,.jpeg"></div>
                 </div>
                 <div class="row responsive-label">
                     <input type="submit" class="primary" value="送信" onclick="return validate();">

